### ISSUE TITLE
WasmNativeWorkload=false doesn't work in wasm runs

### ISSUE THREAD

**Issue by Lxiamail** [dotnet/BenchmarkDotNet#1767]
To avoid wasm_tools to be required in wasm interpreter and wasm aot runs, the WasmNativeWorkload property should be set to false as below. Even though  https://github.com/dotnet/BenchmarkDotNet/blob/f9a4c194b77aa7133168f240b404da58c47d5255/src/BenchmarkDotNet/Templates/WasmCsProj.txt#L16 and https://github.com/dotnet/BenchmarkDotNet/blob/6f453baafa4ef800e0377ebc463ffa82b2f76368/src/BenchmarkDotNet/Templates/WasmAotCsProj.txt#L19 attempted to make the fix, however, it doesn't work for some reason. To temporarily work around this issue, a directory.build.props file had to be inserted to the workitemdirectory of perf run on helix machine, which could have unexpected bad consequences since it is shared by all the perf runs.  Fixing this issue is must-to-have.
<Project>
  <PropertyGroup>
	  <WasmNativeWorkload>false</WasmNativeWorkload>
  </PropertyGroup>
</Project>

**Comment by adamsitnik** [comment]
> however, it doesn't work for some reason

That is very surprising. My current best idea is that there is some `directory.build.props` somewhere in the folder hierarchy that overrides this setting.

**Comment by Lxiamail** [comment]
yes, it is wired that the setting in Wasm*CsProj.txt files didn't work. I will do further testing. Just want to track the issue so that we don't forget.
